### PR TITLE
ELV-145 stop querying for CPU and RAM breakdown by default

### LINKED ISSUE #1883: fmtQueryCPUModePct is costly, consider adding label matchers
**Describe the bug**

This query is being consistently executed: https://github.com/opencost/opencost/blob/55d77f1de56d47ecadd6f3bd38c76e8fb9a4136a/pkg/costmodel/cluster.go#L896. Because it selects `node_cpu_seconds_total` from _all_ servers, it means that it leads to huge load on the metrics stack. Consider adding selectors such as `{cluster="..."}` or something else to make the load smaller.

**To Reproduce**

1. Run OpenCost

**Expected behavior**

OpenCost not pulling in a bunch of data from unrelated servers

**Screenshots**

N/A

**Which version of OpenCost are you using?**

N/A

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by nikovacevic**
## What does this PR change?
* Disables costly Prometheus queries for CPU and RAM mode breakdown. (Allows those features to be re-enabled by helm configuration.)

## Does this PR relate to any other PRs?
* Requires https://github.com/kubecost/cost-analyzer-helm-chart/pull/2448

## How will this PR impact users?
* Less pressure on Prometheus

## Does this PR address any GitHub or Zendesk issues?
* Closes https://github.com/opencost/opencost/issues/1883
* Closes https://kubecost.atlassian.net/browse/ELV-145

## How was this PR tested?
* Manually, using logs and checking API responses

By default:
```
2023-07-19T20:57:12.324097149Z INF ELV-145: skipping node breakdown queries
2023-07-19T20:57:12.334288408Z INF ELV-145: skipping node breakdown queries
2023-07-19T20:57:12.462450038Z INF ELV-145: skipping node breakdown queries
```
Screenshot from 2023-07-19 14-58-24

With the new setting enabled:
```
2023-07-19T21:03:06.363100429Z INF ELV-145: running node breakdown queries
2023-07-19T21:03:06.554459558Z INF ELV-145: running node breakdown queries
2023-07-19T21:03:06.749878635Z INF ELV-145: running node breakdown queries
```
Screenshot from 2023-07-19 15-07-59

## Does this PR require changes to documentation?
* Yes, I'll open a docs PR

### CODE REVIEW

File: pkg/costmodel/cluster.go
Comment (avrodrigues5) [inline pkg/costmodel/cluster.go:319]: clarifying question: Should we check if key is found in the map here Niko?

**Review by avrodrigues5 — APPROVED** [review]
Although this needs an approval from OC maintainer!

File: pkg/costmodel/cluster.go
Comment (nikovacevic) [reply pkg/costmodel/cluster.go:319]: Right above, we ensure it will exist. (Line 307, if it does not exist, then on 308 we create it.) I like your thoroughness, though, and am open to an extra safety check here.

Comment (avrodrigues5) [reply pkg/costmodel/cluster.go:319]: I see in line 307... You good! thanks!

**Comment by nikovacevic** [comment]
Currently, if we merge this and users upgrade, they will panic if they try to roll back to `v1.105` or earlier. This is because older versions do not have `nil` checks in the code, and this change will make those fields `nil` (and this change has the `nil` checks). I think I should probably keep the nil checks, but make the default value empty `&Breakdown{}` and not `nil`, just to be safer.

**Pull request closed without merging**
